src/Chapter06/Listing06.35B.SpecifyingRequiredMembersWithinTheObjectInitializer.cs

```diff
@@ -0,0 +1,27 @@
+#if NET7_0_OR_GREATER
+
+ #if INCLUDE
+using AddisonWesley.Michaelis.EssentialCSharp.Chapter06.Listing06_35A;
+ #endif // INCLUDE
+
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter06.Listing06_35B
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            #if INCLUDE
```

Comment:
Do we really want to use INCLUDE for the symbol, it seems a little too close to our `#region INCLUDE`?

Reply:
@Keboo: How about `#if COMPILEERROR`?

Reply:
That works for me